HousingRepairsSchedulingApi/Startup.cs

```diff
@@ -54,17 +54,16 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddTransient<IDrsService, DrsService>();
 
-            // services.AddTransient<IAppointmentsGateway, DrsAppointmentGateway>(sp =>
-            //     {
-            //         var drsOptions = sp.GetRequiredService<IOptions<DrsOptions>>();
-            //         var appointmentSearchTimeSpanInDays = drsOptions.Value.SearchTimeSpanInDays;
-            //         var appointmentLeadTimeInDays = drsOptions.Value.AppointmentLeadTimeInDays;
-            //         var maximumNumberOfRequests = drsOptions.Value.MaximumNumberOfRequests;
-            //         return new DrsAppointmentGateway(sp.GetService<IDrsService>(),
-            //             5, appointmentSearchTimeSpanInDays, appointmentLeadTimeInDays, maximumNumberOfRequests);
-            //     }
-            // );
-            services.AddTransient<IAppointmentsGateway, DummyAppointmentsGateway>();
+            services.AddTransient<IAppointmentsGateway, DrsAppointmentGateway>(sp =>
+                {
+                    var drsOptions = sp.GetRequiredService<IOptions<DrsOptions>>();
+                    var appointmentSearchTimeSpanInDays = drsOptions.Value.SearchTimeSpanInDays;
+                    var appointmentLeadTimeInDays = drsOptions.Value.AppointmentLeadTimeInDays;
+                    var maximumNumberOfRequests = drsOptions.Value.MaximumNumberOfRequests;
+                    return new DrsAppointmentGateway(sp.GetService<IDrsService>(),
+                        5, appointmentSearchTimeSpanInDays, appointmentLeadTimeInDays, maximumNumberOfRequests);
```

Comment:
What's the significance of '5'? Can this be an enum/explanatory local variable/something other than a magic number?

Reply:
It's the desired number of days worth of appointment.
I've created a constant for it.